botocore/client.py

```diff
@@ -317,4 +349,5 @@ def __init__(self, events):
         self.events = events
 
     def __copy__(self):
-        return ClientMeta(**copy.deepcopy(self.__dict__))
```

Comment:
This was something that @jamesls was also going to do. The event emitter can't be deep copied (things like regexs and partials raise an error when you try), so we have to use a simple copy instead. Unfortunately it means we'll need to maintain this method - anytime you add something to `Meta` you need to make sure it gets copied properly in this method.